spec/default_spec.rb

```diff
@@ -4,7 +4,7 @@
   let(:chef_run) do
     runner = ChefSpec::SoloRunner.new(
       platform: 'debian',
-      version: '7.11'
+      version: '8.11'
```

Comment:
why even support debian 8, I would say 9 and 10 are minimum today?

Reply:
I'll change this when I change the testing for the area 👍 It was just erroring, so I did the minimum to make it work